packages/fscomponents/src/components/CartItem/VariantCartItem.tsx

```diff
@@ -0,0 +1,286 @@
+import { Stepper, StepperProps } from '../Stepper';
+import { Price } from '../Price';
+import i18n, { translationKeys } from '@brandingbrand/fsi18n';
+import { palette } from '../../styles/variables';
+
+const styles = StyleSheet.create({
+  container: {
+    paddingHorizontal: 10,
+    paddingBottom: 30
+  },
+  title: {
+    fontSize: 17,
+    lineHeight: 24,
+    letterSpacing: 0.5,
+    color: palette.onBackground,
+    marginBottom: 15
+  },
+  productImage: {
+    resizeMode: 'contain',
+    width: 132,
+    height: 155
+  },
+  wishlistButton: {
+    marginTop: 7
+  },
+  wishlistButtonText: {
+    fontWeight: '500',
+    fontSize: 13,
+    lineHeight: 15,
+    letterSpacing: 0.5,
+    flexWrap: 'nowrap'
+  },
+  itemWrapper: {
+    flex: 1,
+    flexDirection: 'row'
+  },
+  itemWrapperLeft: {
+    marginRight: 15,
+    minWidth: 135
+  },
+  itemWrapperRight: {
+    flex: 1,
+    justifyContent: 'space-between',
+    paddingBottom: 45
+  },
+  optionValue: {
+    fontWeight: '500'
+  },
+  outOfStockContainer: {
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+    marginTop: 5
+  },
+  outOfStockText: {
+    fontWeight: '500',
+    fontSize: 14,
+    lineHeight: 20,
+    letterSpacing: 0.5,
+    color: palette.error,
+    marginLeft: 5
+  },
+  priceWrapper: {
+    marginBottom: 7
+  },
+  detailLine: {
+    lineHeight: 20,
+    marginBottom: 2
+  }
+});
+
+const icons = {
+  error: require('../../../assets/images/error.png')
+};
+
+export interface SerializableVariantCartItemProps {
+  /**
+   * Additional props that can be applied to the 'move to wishlist' button
+   */
+  wishlistButtonProps?: Partial<ButtonProps>;
+
+  /**
+   * Styles to apply to the main container
+   */
+  style?: ViewStyle;
+
+  /**
+   * Styles to apply to the product item title
+   */
+  titleStyle?: TextStyle;
+
+  /**
+   * Styles to apply to the left column container
+   * (eg. the column with the product image and move to wishlist button)
+   */
+  leftColumnStyle?: ViewStyle;
+
+  /**
+   * Styles to apply to the right column container
+   * (eg. the column with the product details and qty stepper)
+   */
+  rightColumnStyle?: ViewStyle;
+
+  productImageStyle?: ImageStyle;
+  originalPriceStyle?: TextStyle;
+  priceStyle?: TextStyle;
+
+  /**
+   * Text styles that will apply to the price if the item's price is less than originalPrice
+   */
+  salePriceStyle?: TextStyle;
+
+  /**
+   * Text styles that will apply to the item variant detail label (eg. 'Size')
+   */
+  itemDetailLabelStyle?: TextStyle;
+
+  /**
+   * Text styles that will apply to the item variant detail value (eg. 'Small')
+   */
+  itemDetailValueStyle?: TextStyle;
+
+  outOfStockTextStyle?: TextStyle;
+  stepperStyle?: ViewStyle;
+
+  /**
+   * Additional props that will be provided to the stepper
+   */
+  stepperProps?: Partial<StepperProps>;
+}
+
+export interface VariantCartItemProps extends
+  SerializableVariantCartItemProps,
```

Comment:
Could you include StyleProp versions of the style props, so they can take arrays and things like that?